apps/condo/cypress/cypress.config.ts

```diff
@@ -2,19 +2,42 @@ import { defineConfig } from 'cypress'
 
 import conf from '@open-condo/config'
 
-export default defineConfig({
+
+const LOAD_TESTING = conf['CYPRESS_IS_LOAD_TESTING']
+
+const config = {
     e2e: {
         baseUrl: conf['SERVER_URL'],
         projectId: 'dc36jj',
+        video: true,
         videoCompression: 15,
         waitForAnimations: true,
         defaultCommandTimeout: 10000,
+        screenshotOnRunFailure: true,
         retries: 3,
-        setupNodeEvents (on, config) {
-            // eslint-disable-next-line import/order
-            return require('./plugins/index.js')(on, config)
+        setupNodeEvents (cypressOn, config) {
+        // eslint-disable-next-line import/order
+            // https://github.com/cypress-io/cypress/issues/22428
+            const on = require('cypress-on-fix')(cypressOn)
+            require('./plugins/metrics.js')(on, config)
+            require('./plugins/keystone.js')(on, config)
+        },
+        env: {
+            supportPassword: conf['CYPRESS_SERVER_SUPPORT_PASSWORD'],
+            supportEmail: conf['CYPRESS_SERVER_SUPPORT_EMAIL'],
+            grafanaConfig: conf['CYPRESS_GRAFANA_CONFIG'],
         },
         requestTimeout: 10000,
         numTestsKeptInMemory: 1,
     },
-})
+}
+
+if (LOAD_TESTING) {
```

Comment:
It's better to add some arguments to loadtest command inside `package.json`

Reply:
cypress -C ... --config ... does not work apparently :/ so im going to leave it like this for now!